File: SoloLearn/Code Coach/Duty Free/DutyFree.swift
import Foundation

if let prices = readLine() {
    let array = prices.components(separatedBy: " ")
    var max = 0.0
    for price in array {
        if let p = Double(price) {
            if p > max {
                max = p
            }
        }
    }
    if max * 1.1 > 20 {
        print("Back to the store")
    } else {
        print("On to the terminal")
    }
} 

File: SoloLearn/Code Coach/Duty Free/DutyFree.cpp
#include <iostream>
using namespace std;

int main() {
    double price, max = 0.0;
    while (cin >> price) {
        if (price > max)
            max = price;
    }
    if (max * 1.1 > 20)
        cout << "Back to the store";
    else 
        cout << "On to the terminal";
    return 0;
}